tests/kuttl/tests/keystone_scale/03-assert.yaml

```diff
@@ -24,3 +24,11 @@ spec:
   replicas: 3
 status:
   availableReplicas: 3
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: keystone-cron
+spec:
+  schedule: 1 * * * *
+  suspend: false
```

Comment:
nit: I know it's obvious from just reading the file, but could you put the `CronJob` in the list comment at the top?

Reply:
Yes. That makes sense. The line was added in the latest PR.